GiniCapture/Classes/Networking/Extensions/GiniCaptureErrorLogger+GiniCaptureErrorLoggerDelegate.swift

```diff
@@ -7,6 +7,8 @@
 
 import Foundation
 
-extension GiniCaptureErrorLogger {
-   // error.apiLibVerion())
+class GiniErrorLogger: GiniCaptureErrorLoggerDelegate {
```

Comment:
To make it print the error logs you should set an instance of this class in `GiniCapture+GiniCaptureDelegate.swift` to be the `giniErrorLogger`: `configuration.giniErrorLogger = GiniErrorLogger()`.

This way we have the error logging in place and then when the api is ready we just extend this class to send the log to the api.

Reply:
True, we've talked about it yesterday.